client/homescreen/index.js

```diff
@@ -2,37 +2,38 @@
  * External dependencies
  */
 import { compose } from '@wordpress/compose';
-import { Suspense, lazy } from '@wordpress/element';
 import { identity } from 'lodash';
 
 /**
  * WooCommerce dependencies
  */
-import { getSetting } from '@woocommerce/wc-admin-settings';
-import { Spinner } from '@woocommerce/components';
+import { getAdminLink, getSetting } from '@woocommerce/wc-admin-settings';
 import {
 	ONBOARDING_STORE_NAME,
 	withOnboardingHydration,
 } from '@woocommerce/data';
+import { getHistory } from '@woocommerce/navigation';
 
 /**
  * Internal dependencies
  */
 import withSelect from 'wc-api/with-select';
 import { isOnboardingEnabled } from 'dashboard/utils';
 
-const ProfileWizard = lazy( () =>
-	import( /* webpackChunkName: "profile-wizard" */ '../profile-wizard' )
-);
 import Layout from './layout';
 
 const Homescreen = ( { profileItems, query } ) => {
-	if ( isOnboardingEnabled() && ! profileItems.completed ) {
-		return (
-			<Suspense fallback={ <Spinner /> }>
-				<ProfileWizard query={ query } />
-			</Suspense>
+	const {
+		completed: profilerCompleted,
+		skipped: profilerSkipped,
+		step: profilerStep,
+	} = profileItems;
+	if ( isOnboardingEnabled() && ! profilerCompleted && ! profilerSkipped ) {
+		const lastStep = profilerStep ? `&step=${ profilerStep }` : '';
+		const href = getAdminLink(
+			`admin.php?page=wc-admin&path=/profiler${ lastStep }`
 		);
+		getHistory().push( href );
```

Comment:
This one should be updated as well.

Reply:
Thank you @joshuatf, I did the updates in the commit 0a560f2